lms/static/js/dashboard/track_events.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Track the different click events on dashboard.
+ */
+
+var edx = edx || {};
+
+(function ($) {
+    'use strict';
+
+    edx.dashboard = edx.dashboard || {};
+
+    edx.dashboard.TrackEvents = function() {
+        var course_title_link = $(".course-title-link"),
+            course_image_link = $(".dashboard-course-image"),
+            enter_course_link = $(".enter-course-link"),
+            course_action_more = $(".wrapper-action-more"),
+            find_courses_btn = $(".btn-find-courses"),
+            course_learn_verified = $(".learned-verified-track");
+
+        // Fire analytics events when the "course title link" button is clicked.
+        course_title_link.on("click", function (event) {
+            var courseKey = $(event.target).data("course-key");
+            analytics.track(
```

Comment:
@awais786 [`trackLink`](https://segment.com/docs/libraries/analytics.js/#track-link) must be used to guarantee that these events will be emitted. You mentioned that it wasn't working for you. What was the problem? What did you try doing to resolve the issue?